packages/webapp/src/components/ui/ScrollArea.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
```

Comment:
I didn't used it right now but I know the scroll is messed up on windows, it will be used when polishing